src/network/serverpackethandler.cpp

```diff
@@ -716,6 +716,17 @@ void Server::handleCommand_ClientReady(NetworkPacket* pkt)
 			peer_id, major_ver, minor_ver, patch_ver,
 			full_ver);
 
+	std::vector<std::string> players = m_clients.getPlayerNames();
+	for (const std::string &player: m_clients.getPlayerNames()) {
+		NetworkPacket notice_pkt(TOCLIENT_UPDATE_PLAYER_LIST, 0, peer_id);
+		notice_pkt << PLAYER_ADD_INIT <<  player;
+		m_clients.send(peer_id, 0, &notice_pkt, true);
```

Comment:
maybe NetworkPacket can have a helper for sending lists you just serialize the vector heh